src/components/banks/VaultTransaction/index.tsx

```diff
@@ -112,7 +113,9 @@ export const VaultTransaction: FC<Props> = ({
                   <TextInput
                     type="number"
                     value={value}
+                    min={0}
```

Comment:
Have you tried with negative numbers by typing it or scrolling in the mouse?

Reply:
I've tried both